### DISCUSSION TITLE
Map not working in Places view

### DISCUSSION THREAD

**Discussion by djjudas21** [photoprism/photoprism#2096] [Q&A] [answered]
The map in the Places view isn't working for me. It shows markers where the photo locations are, but never loads the background. I've tried setting map type as `Offline` and `Street` but both just load a blank background. Nothing is emitted in the logs. I'm running `220121-2b4c8e1f-Linux-x86_64`.

![Screenshot from 2022-03-02 15-35-29](https://user-images.githubusercontent.com/604595/156421588-29b0618c-f162-482a-85a2-4aaf52c28001.png)

Thanks

**Comment by graciousgrey** [comment, marked as answer]
It's working on our demo: https://demo.photoprism.app/places

So it might be an issue with your setup.

What browser do you use? Do you see errors in the browser console?

Did the maps work before? What's your setup, are you using a proxy like nginx?

**Reply by djjudas21** [reply]
It wasn't working for me on your demo, either. So I restarted my browser (Chrome) and now it's fine. Frustrating, but thanks for your help.

**Reply by pstuifzand** [reply]
I had the same problem, which was fixed by restarting the browser :man_shrugging:

**Comment by rubiojr** [comment]
Learned today that some privacy oriented browsers (like [Librewolf](https://librewolf.net/docs/settings/#enable-webgl)) disable WebGL, which breaks places (can bee seen in the developer console).
